integration/indexes_compat_test.go

```diff
@@ -287,8 +287,8 @@ func TestIndexesCompatCreate(t *testing.T) {
 	}
 }
 
-// TestIndexesCreateRunCommand tests specific behavior for index creation that can be only provided through RunCommand.
-func TestIndexesCompatCreateRunCommand(t *testing.T) {
+// TestIndexesCreateCommandCompat tests specific behavior for index creation that can be only provided through RunCommand.
```

Comment:
Should it be `TestCreateIndexesCommandCompat`?  😀

Reply:
true!